pages/bare_metal_cloud/dedicated_servers/rcw-changing-admin-password-on-windows/guide.en-gb.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "Reset Administrator password when using Rescue-Customer-Windows"
+excerpt: "Reset Administrator password when using Rescue-Customer-Windows"
+updated: 2024-05-21
+---
+
+## Objective
+
+This guide will help you to reset your `Administrator`'s password with the new __*rescue-customer-windows*__
+
+## Requirements
+
+- Microsoft Windows installed on your [dedicated server](/links/bare-metal/bare-metal)
+- At least 16 GB of RAM installed on the server
+- Access to the [OVHcloud Control Panel](/links/manager)
+
+> [!warning]
+>
+> This is not compatible with the `WinPE Rescue`.  
+> Use this [page](/pages/bare_metal_cloud/dedicated_servers/changing-admin-password-on-windows/guide.en-gb.md) if you are using the `WinPe Rescue`.
+>
+
+## Instructions
+
+### Step 1: Rebooting the server into rescue mode <a name="step1"></a>
+
+The system has to be started in rescue mode (WinRescue) before the admin password can be changed.
+
+For detailed instructions, please refer to the rescue mode guide.
+
+
+### Step 2: Clearing the current password <a name="step2"></a>
+
+Connect to your server in Remote Desktop using the credentials provided by email.
+
+Please note that the user name for the rescue is `Administrator`.
+
+If you server have software raid on your Windows disk, you have to import your Windows local disk to be able to access it : follow instructions of [section A](#sectionA)
+
+If you server have no software raid on your Windows disk, you should be able to access directly access the local Windows disk as explained in [section B](#sectionB)
+
+#### A) Import your local Windows disk <a name="sectionA"></a>
+
+1) __Access disk management__
+
+Right click the start menu and select "Disk Management"
+
+![disk_manager_menu](images/disk_manager_menu.png)
+
+You can then see the disks and volumes of the server
+
+![disk_manager_window](images/disk_manager_window1.png)
+
+Your server's Windows disk is probably the disk 1, so you must import it to be able to access it.
+Please note that if you have several disk groups the disk number of Windows may vary, you may have to import several disks to get the Windows one.
+
+2) __Import local disk__
+
+Right click on Disk 1 and select "Online"
```

Comment:
```suggestion
Right-click on Disk 1 and select "Online"
```
I didn't fix them all but the verb has a hyphen https://en.wiktionary.org/wiki/right-click